src/core/Callout/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { AlertProps } from "@material-ui/lab";
+import React from "react";
+import { ReactComponent as IconAlert } from "../../common/svgs/IconAlert.svg";
+import { ReactComponent as IconClose } from "../../common/svgs/IconClose.svg";
+import { ReactComponent as IconSuccess } from "../../common/svgs/IconSuccess.svg";
+import { StyledCallout } from "./style";
+
+export { AlertProps };
```

Comment:
Since the component is Callout, I wonder if we should re-export MUI's `AlertProps` as `CalloutProps`?

E.g,

```ts
import { AlertProps as CalloutProps } from "@material-ui/lab";

export { CalloutProps }
```